etc/kayobe/stackhpc-overcloud-dib.yml

```diff
@@ -21,7 +21,7 @@ stackhpc_overcloud_dib_name: "deployment_image"
 
 # StackHPC overcloud DIB image elements.
 stackhpc_overcloud_dib_elements:
-  - "{{ os_distribution }}-{% if os_distribution == 'rocky' %}container-stackhpc{% else %}minimal{% endif %}"
```

Comment:
I'd consider this as a separate PR to fix RL9 builds

Reply:
Agreed. It's also not fully fixed yet and I don't want to sink more time into it now so I'll put that on hold.